doc/manual/installing.rst

```diff
@@ -34,11 +38,13 @@ Installing multiple packages and making them visible to the ARTIQ commands requi
         pkgs = extrapkg.pkgs;
         artiq = extrapkg.packages.x86_64-linux;
       in {
+        # This section defines the new environment
         defaultPackage.x86_64-linux = pkgs.buildEnv {
```

Comment:
nitpick: update the format to `packages.x86_64-linux.default` from https://github.com/m-labs/artiq/commit/6097a32f4ac34cfb23979590007c4e45a7f3cd11 for consistency